task3/src/Components/Movies/MoviesCount.tsx

```diff
@@ -0,0 +1,11 @@
+import React    from "react";
+
+const MoviesCount : React.FunctionComponent = () => {
```

Comment:
Try to Avoid Export Default
```suggestion
export const MoviesCount : React.FunctionComponent = () => {
```
https://basarat.gitbook.io/typescript/main-1/defaultisbad

Reply:
Remove all 'export default' and fixed imports.